### ISSUE TITLE
MHL_ADM2

### ISSUE THREAD

**Issue by maxmalynowsky** [UGA-ITOSHumanitarianGIS/mapservicedoc#38]
Layer: https://codgis.itos.uga.edu/arcgis/rest/services/COD_NO_GEOM_CHECK/MHL_pcode/FeatureServer/2

Proposed fix: https://data.fieldmaps.io/cod/originals/mhl.shp.zip

Issue: Unknown projection in meters but assigned as WGS84. Manual work done to fix but needs more work to align better with islands as there is still an offset to correct.

<img width="326" alt="Screenshot 2024-04-11 at 14 12 41" src="https://github.com/UGA-ITOSHumanitarianGIS/mapservicedoc/assets/4202166/b0aedafc-0fd1-46b1-ac1e-33764a001cdf">

**Comment by hayttom** [comment]
Is this related to crossing the 180 degree meridian?

**Comment by maxmalynowsky** [comment]
No, it's an issue where the Spatial Reference has been incorrectly assigned. The dataset in HDX has this error too. If the dataset was really EPSG:4326 (WGS84), the Min/Max values should be -180/180 and -90/90 degrees. Here you see values in the millions, which suggests some kind of mercator (meter based) SRS. I can't remember which SRS I used to reassign the data to, I think I used a UTM. It got a ballpark result, but there was still a complex variable offset that I wasn't able to correct.

<img width="1353" alt="Screenshot 2024-05-16 at 11 11 24" src="https://github.com/UGA-ITOSHumanitarianGIS/mapservicedoc/assets/4202166/c3655424-1112-4baa-a113-3a4aa887bb7e">

<img width="1348" alt="Screenshot 2024-05-16 at 11 12 00" src="https://github.com/UGA-ITOSHumanitarianGIS/mapservicedoc/assets/4202166/8b6396c0-f228-453a-bae4-de618d991610">
